bin/testEnvLocalK8S.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
+
+# testEnv will setup a local test environment, for running Istio unit tests.
+
+# Based on circleCI config - used to reproduce the environment and to improve local testing
+
+# expect istio scripts to be under $GOPATH/src/istio.io/istio/bin/...
+export TOP=$(cd $(dirname $0)/../../../..; pwd)
+
+export GOPATH=${TOP}
+export PATH=${GOPATH}/bin:${PATH}
+export OUT=${TOP}/out
+
+# components used in the test (starting with circleci for consistency, eventually ci will use this)
+export K8S_VER=v1.7.4
```

Comment:
This is very old. 1.9 just got released.

Reply:
Using same version that circleci is using, and I believe it's the min. version of k8s we support.

We can run test envs with 1.9 as well.